common/primitives/src/stateful_storage.rs

```diff
@@ -0,0 +1,2 @@
+/// PageId is the unique identifier for a Page in Stateful Storage
+pub type PageId = u8;
```

Comment:
It's less likely that we might have pages more than 256 but I think to future proof this for groups let's change it to `u16`

Reply:
Done.